spec/services/usps_in_person_proofing/enrollment_helper_spec.rb

```diff
@@ -121,7 +121,7 @@
                   }",
                   city: Idp::Constants::MOCK_IDV_APPLICANT_STATE_ID_ADDRESS[:state_id_city],
                   state: Idp::Constants::MOCK_IDV_APPLICANT_STATE_ID_ADDRESS[
-                    :state_id_jurisdiction
+                    :state_id_state
                   ],
```

Comment:
I think this is short enough to fit on one line now rightj?
```
                  state: Idp::Constants::MOCK_IDV_APPLICANT_STATE_ID_ADDRESS[:state_id_state],
```

Reply:
Probably could go on one line, but I'll leave that up to the next person who updates this file.